### ISSUE TITLE
Strings do not work

### ISSUE THREAD

**Issue by EmilyBourne** [pyccel/pyccel#338]
Pyccel complains when trying to use strings as they do not have a precision

**Comment by yguclu** [comment]
Does it complain because `precision=None`, or because the `precision` attribute is missing?

**Comment by EmilyBourne** [comment]
It complains because `precision=None`:
```
File "/home/emily/.local/lib/python3.6/site-packages/pyccel/ast/core.py", line 2267, in __init__
    raise TypeError('precision must be an integer.')
```

**Comment by EmilyBourne** [comment]
This can be fixed in 2 lines in PR #336 once PR #334 is merged

**Comment by yguclu** [comment]
See Epic #459
